sound/hybrid_font.h

```diff
@@ -181,6 +185,17 @@ class HybridFont : public SaberBase {
 	  drag.files_found()) {
 	e = &drag;
       }
+	    
+      // ultra - dynalock lockon
```

Comment:
I don't think we need "ultra" or "dynalock" comments in the base proffieos, it will just confuse things.
A comment like " lockon and lockoff code contributed by YOUR NAME" would be more appropriate.

Reply:
oh you know what, i dont want any credit for it. it just seem silly now. just do what you like with any code you want. its always been intended for that. i hope it helps somehow and im not wasting anyone's time, specially my own. thanks again!